src/directory.rs

```diff
@@ -28,10 +28,19 @@ pub fn scan_directory(
                 )?);
             }
         } else {
-            result.push((
-                previous_commands.join(" "),
-                Script::load(path.as_path().to_str().unwrap_or_default())?,
-            ));
+            if script::is_tome_script(
+                // path.file_name().to_str().unwrap_or_default(),
+                path.file_name()
+                    .unwrap_or_default()
+                    .to_str()
+                    .unwrap_or_default(),
+                )
```

Comment:
could this be moved out to a variable then passed in as an argument, so we don't have to indent so heavily? would make the lines a bit easier to read?

Reply:
done, and collapsed to a `else if` block on the recommendation of `cargo clippy`